internal/sops/keyservice/server.go

```diff
@@ -96,6 +102,16 @@ func (ks Server) Encrypt(ctx context.Context, req *keyservice.EncryptRequest) (*
 				Ciphertext: ciphertext,
 			}, nil
 		}
+	case *keyservice.Key_KmsKey:
+		if ks.awsCreds != nil {
```

Comment:
Same [suggestion as for GCP KMS](https://github.com/fluxcd/kustomize-controller/pull/635#discussion_r856353043).